### DISCUSSION TITLE
Joint JS Paper Scrollable or Stretchable

### DISCUSSION THREAD

**Discussion by adarshjalaja** [clientIO/joint#2232] [Q&A]
### Introduction

We are working on an application where we would like the user to stretch paper and add more and more elements inside
or make paper scrollable to any extent to draw innumerable elements

is this doable with Joint js (non commercial version)

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS

**Comment by kumilingus** [comment]
Everything that you see in _JointJS+_ is built on top of _Open Source JointJS_ using its API.

- You can change the size of the paper with `paper.setDimensions()`  or  `paper.fitToContent()` anytime the graph changes (specifically on cell `add` and  `remove`, on `position`, `size`, `angle` change).
- You can insert the `paper.el` into a container with `overflow: scroll`.

**Reply by adarshjalaja** [reply]
Thanks for quick reply . appreciate
